config.json

```diff
@@ -203,7 +201,7 @@
     0,
     0
   ],
-  "SaveName": "Arkham SCE - 3.0",
+  "SaveName": "Arkham SCED - 3.0.0",
```

Comment:
Is this name change intended? Savegames have usually been called "SCE"
![Arkham SCE 2 4 0](https://user-images.githubusercontent.com/97286811/214798137-6d9a3214-413e-473c-848e-38dc74e9dfd7.png)

Reply:
Oh, good catch! Thank you